### PR TITLE
Add kytos controller commands

### PR DESCRIPTION
**Pull request by macartur**

### CODE REVIEW

File: kytos/cli/commands/controller/parser.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+       kytos controller start
+       kytos controller stop
+       kytos controller restart
+       kytos controller -h | --help
+
+Options:
+
+  -h, --help    Show this screen.
+
+Common napps subcommands:
```
Comment (diraol) [inline kytos/cli/commands/controller/parser.py]: `Common controller sub commands`

File: kytos/cli/commands/controller/parser.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  start        Run kytos controller in daemon context.
+  stop         Stop kytos controller in daemon context.
```
Comment (diraol) [inline kytos/cli/commands/controller/parser.py]: `stop` works only if kytos is running in daemon context? What if kytos is running in foreground?

File: kytos/cli/commands/controller/parser.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  restart      Stop and restart kytos controller in daemon context.
```
Comment (diraol) [inline kytos/cli/commands/controller/parser.py]: Same question from `stop` here.

File: kytos/cli/commands/controller/api.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import subprocess
+
+
+class ControllerAPI:
+    """Handle KytosController comamnds."""
+
+    @staticmethod
+    def start():
+        """Start KytosController."""
+        subprocess.run(["kytosd"])
```
Comment (beraldoleal) [inline kytos/cli/commands/controller/api.py]: Did you tried to run this command, when kytosd is already runing ?

Comment (beraldoleal) [reply kytos/cli/commands/controller/api.py]: Besides that, kytos-utils is made to run in a different controller. What happen when kytosd is running in a different server ?

Comment (macartur) [reply kytos/cli/commands/controller/api.py]: This PR only works in local server. If kytosd is called when another kytosd is already running the message below will be displayed.

`The process is already running with pid 24702.`

**Pull request merged**